readthedocs/settings/docker_compose.py

```diff
@@ -22,7 +22,9 @@ class DockerBaseSettings(CommunityDevSettings):
     SLUMBER_API_HOST = 'http://web:8000'
     RTD_EXTERNAL_VERSION_DOMAIN = 'org.dev.readthedocs.build'
 
-    STATIC_URL = f'http://{PRODUCTION_DOMAIN}/devstoreaccount1/static/'
+    # storage:10000 is running the Azure Blob Storage and it's exposed to the
+    # host on the same port that user's can access via localhost
+    STATIC_URL = f'http://localhost:10000/devstoreaccount1/static/'
```

Comment:
Should it continue to use `PRODUCTION_DOMAIN`, but on port 10000? That seems a bit cleaner, and should still resolve to localhost.

Reply:
Yeah. I think you are right. I may messed things up here.

I will update this PR with latest master and see if it still works :)

Reply:
So, yes. Although, the way to specify this is with our `AZURE_STATIC_STORAGE_HOSTNAME` that it's used to override the domain returned by `storage.url` in our Mixin.

This STATIC_URL just need to be `/devstorageaccount1/static/`

BTW, I'm using `assets.community.dev.readthedocs.io` so it's a different domain and it follows the same idea than production.